app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -171,8 +171,23 @@ class EnabledContent extends ImmutableComponent {
   render () {
     const ledgerData = this.props.ledgerData
     const walletStatusText = walletStatus(ledgerData)
-
-    return <section>
+    const inTranstion = !(ledgerData.getIn(['migration', 'btc2BatTransitionDone']))
+
+    return <section className={css(styles.enabledContent)}>
+      <div className={css(styles.loader, inTranstion && styles.loader_show)}>
+        <p className={css(styles.loader__text)}>
+          Welcome to the new Payment page integrated with Basic Attention Token (BAT)<br />
+          We are converting your old BTC wallet to the new and shiny BAT wallet. <br />
+          This can take a few minutes
+        </p>
+        <div className={css(styles.leader__wrap)}>
+          <div>
+            <div className={css(styles.loader__line, styles.loader__line_1)} />
+            <div className={css(styles.loader__line, styles.loader__line_2)} />
+            <div className={css(styles.loader__line, styles.loader__line_3)} />
+          </div>
+        </div>
+      </div>
       <div className={css(styles.walletBar)} data-test-id='walletBar'>
```

Comment:
`styles.walletBar` -> `styles.enabledContent__walletBar`

Reply:
this is not part of this PR

Reply:
This change requires that: https://github.com/brave/browser-laptop/pull/11395/files#diff-5d8199641ea1ba12e0394859f7eb0c55R175